src/constants.ts

```diff
@@ -190,3 +191,200 @@ export const SCOPES_BY_TITLE = keyBy(
+      '[Consent Management -> Developer Settings -> Advanced Settings]' +
+      '(https://app.transcend.io/consent-manager/developer-settings/advanced-settings)',
+  },
+  [TranscendPullResource.Prompts]: {
+    description: 'The Transcend AI prompts',
+    markdownLink:
+      '[Prompt Manager -> Browse](https://app.transcend.io/prompts/browse)',
+  },
+  [TranscendPullResource.PromptPartials]: {
+    description: 'The Transcend AI prompt partials',
+    markdownLink:
+      '[Prompt Manager -> Partials](https://app.transcend.io/prompts/partialss)',
+  },
+  [TranscendPullResource.PromptGroups]: {
+    description: 'The Transcend AI prompt groups',
+    markdownLink:
+      '[Prompt Manager -> Groups](https://app.transcend.io/prompts/groups)',
+  },
+  [TranscendPullResource.Agents]: {
+    description: 'The agents in the prompt manager.',
+    markdownLink:
+      '[Prompt Manager -> Agents](https://app.transcend.io/prompts/agents)',
+  },
+  [TranscendPullResource.AgentFunctions]: {
+    description: 'The agent functions in the prompt manager.',
+    markdownLink:
+      '[Prompt Manager -> Agent Functions](https://app.transcend.io/prompts/agent-functions)',
+  },
+  [TranscendPullResource.AgentFiles]: {
+    description: 'The agent files in the prompt manager.',
+    markdownLink:
+      '[Prompt Manager -> Agent Files](https://app.transcend.io/prompts/agent-files)',
+  },
+  [TranscendPullResource.Vendors]: {
+    description: 'The vendors in the data inventory.',
+    markdownLink:
+      '[Data Inventory -> Vendors](https://app.transcend.io/data-map/data-inventory/vendors)',
+  },
+  [TranscendPullResource.DataCategories]: {
+    description: 'The data categories in the data inventory.',
+    markdownLink:
+      '[Data Inventory -> Data Categories](https://app.transcend.io/data-map/data-inventory/data-categories)',
+  },
+  [TranscendPullResource.ProcessingPurposes]: {
+    description: 'The processing purposes in the data inventory.',
+    markdownLink:
+      '[Data Inventory -> Processing Purposes](https://app.transcend.io/data-map/data-inventory/purposes)',
+  },
+  [TranscendPullResource.ActionItems]: {
+    description: 'Onboarding related action items',
+    markdownLink: '[Action Items](https://app.transcend.io/action-items/all)',
+  },
+  [TranscendPullResource.ActionItemCollections]: {
+    description: 'Onboarding related action item group names',
+    markdownLink: '[Action Items](https://app.transcend.io/action-items/all)',
+  },
+  [TranscendPullResource.Teams]: {
+    description: 'Team definitions of users and scope groupings',
+    markdownLink:
+      '[Administration -> Teams](https://app.transcend.io/admin/teams)',
+  },
+  [TranscendPullResource.PrivacyCenters]: {
+    description: 'The privacy center configurations.',
+    markdownLink:
+      '[Privacy Center](https://app.transcend.io/privacy-center/general-settings)',
+  },
+  [TranscendPullResource.Policies]: {
+    description: 'The privacy center policies.',
```

Comment:
```suggestion
    description: 'The Privacy Center configurations.',
    markdownLink:
      '[Privacy Center](https://app.transcend.io/privacy-center/general-settings)',
  },
  [TranscendPullResource.Policies]: {
    description: 'The Privacy Center policies.',
```

_also a port but now i'm re-reading it!_